src/Template/Element/sentences/sentence_and_translations.ctp

```diff
@@ -110,7 +119,8 @@ $sentenceUrl = $this->Url->build([
             <?php
             if (CurrentUser::isMember()) {
                 echo $this->element('sentences/sentence_menu', [
-                    'expanded' => $menuExpanded
+                    'expanded' => $menuExpanded,
+                    'sentenceData' => $sentenceData
```

Comment:
Isn’t this change unnecessary?

Reply:
Yes you're right. That's a leftover from the old version that I forgot to remove. Thanks.